notebooks/data-sources/gcsweb-ci/build-logs/model_seldon.ipynb

```diff
@@ -0,0 +1,562 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    # Check if your model exists on s3
</pre><p>Can we uncomment this line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
it will be useful to check if the model is present on s3 as a sanity check